kubernetes/.swagger-codegen/VERSION

```diff
@@ -0,0 +1 @@
+2.3.0-SNAPSHOT
```

Comment:
Specific commit or release version sounds best - how were you going about pointing at the specific commit? If that can be set externally to the pom.xml, pointing at -SNAPSHOT will use the codegen at that commit.